src/Path.hs

```diff
@@ -520,6 +523,9 @@ normalizeFilePath = normalizeLeadingSeparators . FilePath.normalise
 --------------------------------------------------------------------------------
 -- Deprecated
 
+{-# DEPRECATED PathParseException "Please use PathException instead." #-}
+type PathParseException = PathException
```

Comment:
Add doc-string to this one and Cabal builds should pass. We catch not-100% Haddock coverage, not sure why it's so unfriendly though (my typical setup prints per-module coverages).